File: Week 1 - 03.02.2021/Bubble Sort 46.cpp
// Bubble Sort using C++
// Author : Abhishek Sharma

#include<iostream>
using namespace std;
 
int main()
{
 int N,i,j,a[100],c=0;
 cin>>N;
 for(i=0;i<N;i++)
 {
     cin>>a[i];
 }
 for(j=0;j<N-1;j++)
 {
     for(i=0;i<N-j-1;i++)
     {
         if(a[i]>a[i+1])
         {
             a[i]=a[i]+a[i+1];
             a[i+1]=a[i]-a[i+1];
             a[i]=a[i]-a[i+1];
             c++;
         }
     }
 }
 
cout<<c;
    return 0;
}

File: Week 1 - 03.02.2021/Bubble Sort 46.py
# Bubble Sort using Python
# Author : Abhishek Sharma

n = int(input().strip())
user= list(map(int, input().split()))
           
count = 0
for j in range (0,n-1):
    for i in range (0,n-j-1):
        if user[i]>user[i+1]:
            user[i]=user[i]+user[i+1]
            user[i+1]=user[i]-user[i+1]
            user[i]=user[i]-user[i+1]
            count = count + 1
            
print (count)
